Consul Connect (#64)
hashicorp/katakoda: consul-connect/intro.md

# Consul Connect Service Mesh

Consul service mesh secures service-to-service communication with authorization and encryption. Applications can use sidecar proxies in a service mesh configuration to automatically establish TLS connections for inbound and outbound connections without being aware of the network configuration and topology. 

In this hands-on lab, you'll start two services and connect them over a TLS encrypted proxy with Consul service mesh. 

Specifically, you'll start four processes.

> - The frontend service is be a dashboard that displays a number. 
> - The backend service is be a counting application that serves a JSON feed with a constantly incrementing number.

Reviewer: ```suggestion
- The frontend service is a dashboard that displays a number. 
- The backend service is a counting application that serves a JSON feed with a constantly incrementing number.
```

- Two `consul connect` sidecar proxies represent the `dashboard` and `counting` services so they can communicate securely with each other.

> The frontend service uses websockets to update its user interface every few seconds with fresh data from the backend service. It also displays status information so you can see when the connection has been made.

Reviewer: ```suggestion
The frontend service uses websockets to update its user interface every few seconds with fresh data from the backend service. It also displays status information so you can see if the connection can be established.
```
